infrastructures/kubernetes/pom.xml

```diff
@@ -66,6 +66,10 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
```

Comment:
I guess we don't have CQ for this dependency. Can you use some alternatives?

Reply:
@skabashnyuk seems to be defined in parent https://github.com/eclipse/che-parent/blob/master/pom.xml#L252

Reply:
It's not an issue where it's defined. the problem is that it becomes compile level and we have no cq for it. https://dev.eclipse.org/ipzilla/buglist.cgi?query_format=advanced&short_desc_type=allwordssubstr&short_desc=&component=ecd.che&long_desc_type=substring&long_desc=&bug_file_loc_type=allwordssubstr&bug_file_loc=&keywords_type=allwords&keywords=&emailassigned_to1=1&emailtype1=substring&email1=&emailassigned_to2=1&emailreporter2=1&emailcc2=1&emailtype2=substring&email2=&bugidtype=include&bug_id=&chfieldfrom=&chfieldto=Now&chfieldvalue=&cmdtype=doit&order=Reuse+same+sort+as+last+time&field0-0-0=noop&type0-0-0=noop&value0-0-0=

AFAIK you can find the same helper method on other artifacts like guava

Reply:
@skabashnyuk ok, good to know - changed to `com.google.common.base.Strings